ntbs-build/database-deploy-scripts/2_Deploy_Databases.sh

```diff
@@ -0,0 +1,78 @@
+DEPLOY_ENVIRONMENT=${DEPLOY_ENVIRONMENT:-uat}
+
+if [[ $DEPLOY_ENVIRONMENT = "live" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Production\PHE-LIVE ntbs-data-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\DELETE\phe-ntbs-live-reporting-DELETE.publish.xml"
+    specimen_matching_publish_profile=".\source\phe-ntbs-live-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "uat" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Delete\PHE-UAT ntbs-data-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\DELETE\phe-ntbs-uat-reporting-DELETE.publish.xml"
+    specimen_matching_publish_profile=".\source\phe-ntbs-uat-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "training" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Delete\DELETE NTBS-TRAINING ntbs-data-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\DELETE\azure-ntbs-training-reporting-DELETE.publish.xml"
+    specimen_matching_publish_profile=".\source\azure-ntbs-training-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "int" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Delete\DELETE NTBS-DEV ntbs-data-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\DELETE\azure-ntbs-int-reporting-DELETE.publish.xml"
+    specimen_matching_publish_profile=".\source\azure-ntbs-int-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "dev" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Delete\DELETE DEV ntbs-data-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\DELETE\DEV-reporting-DELETE.publish.xml"
+    specimen_matching_publish_profile=".\source\DEV-specimen-matching.publish.xml"
+else
+    echo "Invalid environment entered."
+    exit 1
+fi
+
+read -p "Are you running this script on the PHE dev box? (default yes): " ON_DEV_BOX
+ON_DEV_BOX=${ON_DEV_BOX:-yes}
+
+if [[ $ON_DEV_BOX = "yes" ]]
+then
+    MSBUILD_PATH="C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe"
+    SQLPACKAGE_PATH="C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\Extensions\Microsoft\SQLDB\DAC\150\sqlpackage.exe"
+else
+    # This is where these executables are stored on my machine - other users will need to check/update these if running the script locally
+    MSBUILD_PATH="C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Current\Bin\MSBuild.exe"
+    SQLPACKAGE_PATH="C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\Extensions\Microsoft\SQLDB\DAC\150\sqlpackage.exe"
+fi
+
+deploy_database() {
+    echo -e "\nDeploying $1 to database...\n"
+    cd "./$1"
+    "$MSBUILD_PATH" "$2"
+    "$SQLPACKAGE_PATH" //Action:Publish //Profile:"$3" //SourceFile:"$4"
```

Comment:
Do we want DELETE script functionality? It does feel a little bit risky though

Reply:
I think we do, because otherwise we're not releasing in a truly reproducible way. It is potentially a little risky, but since the data is all truncated and rebuilt regularly I don't think it's a massive problem. What kind of scenarios are you worried about?

Reply:
Mainly just pressing the wrong button! I hope that would be a bit easier to control for when using an OpenShift pipeline instead of a shell script though

Reply:
Fair enough. Since these scripts are temporary I'll leave this as-is. If/when this is a pipeline/action that we using for more than just releases (i.e. if we're using it to deploy to the dev/azure environments) then for those environments it may well be better to use the normal profile and say that if someone wants to use the delete profile then they need to do so manually.

Reply:
Yeah that sounds good!